src/stravalib/client.py

```diff
@@ -297,7 +297,10 @@ def get_activities(
         after_epoch = self._utc_datetime_to_epoch(after) if after else None
         params = dict(before=before_epoch, after=after_epoch)
         result_fetcher = functools.partial(
-            self.protocol.get, "/athlete/activities", **params
+            self.protocol.get,
+            "/athlete/activities",
+            check_for_errors=True,
```

Comment:
This is silly. mypy is erroring because it expects a bool. but True is a default for check_for_errors. but what it sees is a dictionary to unpack as the second param provided to the get method. 

so i just added it here even tho it's redundant 🤷🏻‍♀️ if there is a better preferred way please let me know!